src/ga/layer/layer.js

```diff
@@ -279,8 +279,8 @@ ga.source.wmts = function(layer, options) {
         '" target="new">' +
         options['attribution'] + '</a>')
     ],
-    url: ('http://wmts{5-9}.geo.admin.ch/1.0.0/{Layer}/default/{Time}/21781/' +
-        '{TileMatrix}/{TileRow}/{TileCol}.').replace('http:',location.protocol) + extension,
+    url: ('http://tod{0-4}.prod.bgdi.ch/1.0.0/{Layer}/default/{Time}/2056/' +
```

Comment:
@gjn what urls do we plan to use for the go live of lv95 ? wmts{5-9} as before ?